load_clinical.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+echo "Directory Name for study (ex: Neptune_V16): "
```

Comment:
Would it be useful to have a command format reminder if someone tries to run the script with no parameters (or the wrong number of parameters)?

Reply:
No, since we are prompting them for the information, not asking for them as parameters.